### PR TITLE
Chore: Update README to reflect to product

### PR DESCRIPTION
**Pull request by cjellick**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,37 +1,80 @@
...
+
+#### Working with Local Files
+
+[Knowledge](https://github.com/gptscript-ai/knowledge), [Structured Data Querier](https://github.com/gptscript-ai/structured-data-querier), [Filesystem](https://github.com/gptscript-ai/context/tree/main/filesystem), [Workspace](https://github.com/gptscript-ai/context/tree/main/workspace), [PDF OCR Reader](https://github.com/gptscript-ai/pdf-tool/tree/main/gateway)
+
+#### Coding and DevOps
+
+[AWS](https://github.com/gptscript-ai/tools/tree/main/clis/aws), [Azure](https://github.com/gptscript-ai/tools/tree/main/clis/azure), [Digital Ocean](https://github.com/gptscript-ai/tools/tree/main/clis/digitalocean), [EKS](https://github.com/gptscript-ai/tools/tree/main/clis/eksctl), [GCP](https://github.com/gptscript-ai/tools/tree/main/clis/gcp), [Github](https://github.com/gptscript-ai/tools/tree/main/clis/github), [k8s](https://github.com/gptscript-ai/tools/tree/main/clis/k8s), [Supabase cli](https://github.com/gptscript-ai/tools/tree/main/clis/supabase), [Supabase API](https://github.com/gptscript-ai/tools/tree/main/apis/supabase)
```
Comment (drpebcak) [inline README.md:39]:
```suggestion
[AWS](https://github.com/gptscript-ai/tools/tree/main/clis/aws), [Azure](https://github.com/gptscript-ai/tools/tree/main/clis/azure), [DigitalOcean](https://github.com/gptscript-ai/tools/tree/main/clis/digitalocean), [EKS](https://github.com/gptscript-ai/tools/tree/main/clis/eksctl), [GCP](https://github.com/gptscript-ai/tools/tree/main/clis/gcp), [GitHub](https://github.com/gptscript-ai/tools/tree/main/clis/github), [k8s](https://github.com/gptscript-ai/tools/tree/main/clis/k8s), [Supabase cli](https://github.com/gptscript-ai/tools/tree/main/clis/supabase), [Supabase API](https://github.com/gptscript-ai/tools/tree/main/apis/supabase)
```

File: README.md [hunk 2]
```diff
@@ -1,37 +1,80 @@
-# ui
-This provides a graphical interface to run and build your GPTScripts.
+# Acorn Desktop
 
-![Chat Image](assets/chat.png)
+💫 **The best way to build, use, and share AI Assistants.** 💫 
 
-## Running
-The UI is baked directly into the [gptscript](https://docs.gptscript.ai/) CLI. To run the UI, simply run the following command:
+https://github.com/user-attachments/assets/b9cfa1c3-ba76-4193-84bf-115096b1a3ee
```
Comment (drpebcak) [inline README.md:5]: What is this link for?

File: README.md [hunk 3]
```diff
@@ -1,37 +1,80 @@
...
 
-You can also specify a specific script to run (locally or remote).
+**Tools** are Acorn Desktop's super power. They let the user and assistants interact with the rest of the world, including the user's workstation, APIs, and the Internet.
 
-```bash
-gptscript --ui github.com/gptscript-ai/llm-basics-demo
-```
+**Assistants** bring tools and the user together. They couple tools with context that affects its behavior, focus, and even personality.
+
+A tool is a self-contained piece of functionality that an assistant can use to accomplish the tasks that a user lays out for it. Tools themselves can be LLM instructions, code-based (in python, node.js, or bash), or integrations with external systems.
```
Comment (drpebcak) [inline README.md:13]: Worth mentioning that a tool can be go as well?

**Review by drpebcak — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (cjellick) [reply README.md:5]: video recording

File: README.md (hunk 3 above)
Comment (cjellick) [reply README.md:13]: in the context of this app, no bc this is referring to the inline tools you can create and those cannot be go

**Comment by cjellick** [comment]
Thanks for the review @drpebcak. I'm going to go ahead and merge

File: README.md (hunk 2 above)
Comment (drpebcak) [reply README.md:5]: Ah, it doesn't work in the mobile app for some reason, but I see it on desktop

**Pull request merged**